Code/GraphMol/Aromaticity.cpp

```diff
@@ -701,7 +807,10 @@ int aromaticityHelper(RWMol &mol, unsigned int minRingSize,
     for (unsigned int ri = 0; ri < cRings.size(); ++ri) {
```

Comment:
might as well change to 
```
for (auto ri : crings) 
```

?

Reply:
Nope, we use the index, not the ring itself here.